app/hero/Hero.tsx

```diff
@@ -1,23 +1,34 @@
-import { Heading, Wrap, WrapItem, Text, Box } from "@chakra-ui/react";
+import { Button, Heading, Wrap, WrapItem, Text, Box } from "@chakra-ui/react";
 import { memo } from "react";
 import { Image } from "~app/image";
 import backgroundImage from "./bg.jpg";
 
 export default memo(function Hero() {
 	return (
-		<Wrap spacing={8} justify="center" align="center" textAlign="center">
-			<WrapItem flex="1 1 0">
-				<Box>
-					<Heading as="h1" size="3xl" d="block">
-						Louise-Schroeder-Gymnasium
+		<Wrap
+			spacing={8}
+			justify="center"
+			align="center"
+			textAlign="center"
+			maxW="6xl">
```

Comment:
This is a part of the page content, which should have a uniform `max-width` wrapper. No need to apply it here.